chapters/methodology.tex

```diff
@@ -9,8 +9,20 @@ \subsection{Architecture}
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
 \subsection{State Machine}
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
-\subsection{User Interface}
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
+\begin{figure}
+  \includegraphics[width=\linewidth]{figures/state_diagramm.jpg}
+  \caption{The state diagram.}
+  \label{fig:state_diagramm}
+\end{figure}
+
 
+Figure \ref{fig:state_diagramm} shows the states and transitions of the state machine.
+A state change can be invoked throw three different kind of actions: An user action (for example pressing a button), a timeout after a specified couple of seconds or an significant analyser result (face detection or smile detection).
+
+Note that if a face is detected and the user moves his face out of the frame afterwards, the process continues and handles this event indirectly as missing smile.
+Consistently, the face leaving the camera frame after a first smile detection is not dealed with explicitly but leads to low smile results.

```

Comment:
Not sure whether consistently is the right word here. Are you trying to emphasize the contrast between this statement and the previous one?

Reply:
I'm trying to show the simultaneity...

Reply:
Idk if this helps... If you like to point out the simultaneity, I'd use Moreover / Furthermore / in addition